fenix/app/src/main/java/org/mozilla/fenix/compose/button/FloatingActionButton.kt

```diff
@@ -43,13 +46,15 @@ fun FloatingActionButton(
     modifier: Modifier = Modifier,
     contentDescription: String? = null,
     label: String? = null,
+    elevation: FloatingActionButtonElevation = FloatingActionButtonDefaults.elevation(defaultElevation = 5.dp),
```

Comment:
I do see in the figma that we have 3 values 5, 18, 10 dp which probably represents the other states. I know we only care about the default state from the ticket, but is there any reason we are not adjusting the elevation for the hovered, pressed and focused state while we are already adjusting the elevation here?

I think the default values for the other state are fine, but unclear why we are excluding the elevation.

Reply:
I'll be honest, I have no idea how Figma is deriving the 18 and 10 values in the Compose code generator tool. I got the single value of 5 by checking the XML code.

As far as I can tell, the design system doesn't have a language for these other states, so I just wanted to get this change landed and defer that conversation for now, as we have no precedence for specifically setting the elevation (or colors) for pressed and hovered states.

Reply:
sgtm